src/psyclone/tests/psyir/nodes/array_mixin_test.py

```diff
@@ -431,6 +436,14 @@ def test_get_bound_expression():
     assert ub2 is not ubnd_ref
     assert ub2 == ubnd_ref
 
+    # An ArrayType where the upper bound is unknown.
+    symbol2 = DataSymbol("other", ArrayType(INTEGER_TYPE,
+                                            [(1, ArrayType.Extent.ATTRIBUTE)]))
```

Comment:
Reading the ArrayType docstring this is not a valid shape, it should be just `[ArrayType.Extent.ATTRIBUTE]`

Reply:
I've updated the docstring.